translations/resources.tex/pl.po

```diff
@@ -68,6 +68,23 @@ msgid ""
 "  \\includegraphics[width=0.8\\linewidth]{\\images/valuables.png}\n"
 "  \\footnotesize\\textcolor{darkcandyapplered}{\\textit{\\textbf{Valuables \\phantom{Materials}}}}"
 msgstr ""
+"\\hypertarget{Resources}{W grze} istnieją trzy rodzaje \\index{Zasoby} zasobów: Złoto \\includesvg[height=10px]{\\svgs/gold.svg}, Budulce \\includesvg[height=10px]{\\svgs/building_materials.svg} oraz Kosztowności \\includesvg[height=10px]{\\svgs/valuablegreater.svg}.\n"
+"Zasoby będziesz wydawał w trakcie gry na rozbudowę \\hyperlink{Town}{Miasta}, Rekrutację \\hyperlink{Units}{Jednostek} i zakup \\hyperlink{spells}{Zaklęć}.\n"
+"Zazwyczaj będziesz je pozyskiwał z \\hyperlink{Mines}{Kopalń oraz Osad}, które \\hyperlink{Categories}{Oflagowałeś}, a także zagrywając Karty i rzucając Kośćmi Zasobów \\includesvg[height=10px]{\\svgs/resource_die.svg}.\n"
```

Comment:
nie będę się upierał, ale dla mnie to wygląda nienaturalnie (choć jest poprawne i to i to), ale załączam też reference z oryginału :P

```suggestion
"Zazwyczaj będziesz je pozyskiwał z \\hyperlink{Mines}{Kopalni oraz Osad}, które \\hyperlink{Categories}{Oflagowałeś}, a także zagrywając Karty i rzucając Kośćmi Zasobów \\includesvg[height=10px]{\\svgs/resource_die.svg}.\n"
```
<img width="299" alt="Screenshot 2024-04-30 at 11 37 05" src="https://github.com/Heegu-sama/Homm3BG/assets/9850720/e78a34e7-3c87-4fca-a766-1c1725ea2a32">

Reply:
Poprawione